docs/framework/20categories/30implementation/verified-code.md

```diff
@@ -1,3 +1,19 @@
 ### Verified Code on Block Explorer
 
-The source code for deployed contracts should be verified and made available in the network's primary block explorer (e.g., EtherScan for Ethereum, SnowTrace for Avalanche). Not being able to determine what code has been deployed reduces the trust stakeholders have in the project. 
\ No newline at end of file
+Source code verification for all deployed smart contracts is critical for user safety. It acts as an assurance of correctness of a smart contract by ensuring that a protocol’s submitted source code is the same as the one executed at the contract address. Moreover, it allows users interacting with a protocol’s smart contract to read and audit it. This enables them to determine what code has been deployed and what types of risks the user is taking by interacting with the protocol. As a result, if a protocol’s code isn’t verified, it is considered less transparent and should be trusted less by users and developers. 
```

Comment:
Should we attempt to rephrase the second sentence for clarity:
- I fear that the "correctness of a smart contract" can be a misleading phrase since correctness often refers to functional correctness
- the phrase "a protocol's submitted code" sounds a tad unclear. do we mean their open-sourced repository version?